.github/workflow-scripts/gh-pages/deploy.js

```diff
@@ -75,7 +75,18 @@ async function deploy({ branch, sha, repo, owner, docsPath, github, exec }) {
     if (fs.existsSync(docsPath)) {
       // this is to avoid any merge conflict when an environment is redeployed
       console.info(`ℹ️ Cleaning existing ${docsPath}`);
-      await exec.exec('rm', ['-rf', docsPath]);
+      if (branch === 'master') {
+        // for master we want to clean everything from `./docs` with the exception of the directories of the deployed branches
+        const filesToRemove = fs.readdirSync('./docs', { withFileTypes: true }).filter(item => !item.name.startsWith('sb-')).map(({ name }) => name);
```

Comment:
I don't have a huge problem with it, but it does seem like this file has an identity crisis with using node for reading files and shell scripts for moving them :)

Reply:
Please don't make me change it 🤣